src/Input/Input.js

```diff
@@ -323,9 +368,16 @@ module.exports = kind(
 	/**
 	* @private
 	*/
-	input: function () {
+	input: function (sender, e) {
 		var val = this.getNodeProperty('value');
 		this.setValue(val);
+		if(this._enabledValidity) {
```

Comment:
I would suggest to make enableValidity as published property and use here. So, user can explicitly set it as true when he want to enable validity check feature.